src/Entity/VotingPlatform/Designation/Designation.php

```diff
@@ -66,7 +117,10 @@ class Designation implements EntityAdministratorBlameableInterface
      *
      * @ORM\Column
      *
-     * @Assert\NotBlank
+     * @Assert\NotBlank(groups={"Default", "api_designation_write"})
+     * @Assert\Choice(choices=DesignationTypeEnum::MAIN_TYPES, strict=true, groups={"Default", "api_designation_write"})
```

Comment:
I think we should limit type collection for API context, `API_AVAILABLE_TYPES` will contain only `COMMITTEE` in the beginning
```suggestion
     * @Assert\Choice(choices=DesignationTypeEnum::MAIN_TYPES, strict=true, groups={"Default"})
     * @Assert\Choice(choices=DesignationTypeEnum::API_AVAILABLE_TYPES, strict=true, groups={"api_designation_write"})
```